webui/react/src/e2e/models/components/GridListRadioGroup.ts

```diff
@@ -0,0 +1,12 @@
+import { BaseComponent } from 'playwright-page-model-base/BaseComponent';
+
+export class GridListRadioGroup extends BaseComponent {
```

Comment:
change this to named component with defaultSelector = `[data-testid="grid-list-radio-group"]`